recipe/build.sh

```diff
@@ -3,9 +3,24 @@
 # Install to conda style directories
 [[ -d lib64 ]] && mv lib64 lib
 
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
 for i in `ls`; do
     [[ $i == "build_env_setup.sh" ]] && continue
     [[ $i == "conda_build.sh" ]] && continue
     [[ $i == "metadata_conda_debug.yaml" ]] && continue
+
+    if [[ $i == "bin" ]]; then
+        for j in `ls "${i}"`; do
+            [[ -f "bin/${j}" ]] || continue
+            [[ ${j} == "cuda-gdb" && ${target_platform} == "linux-64" ]] && continue # cuda-gdb is a shell script for linux-64
```

Comment:
Thanks for catching this Billy! 🙏

Just to clarify for anyone who reads this thread...

* `cuda-gdb` is a binary on `linux-aarch64`
* `cuda-gdb` is a shell script on `linux-64`

So we patch the former and skip the latter